test/cases/api.sh

```diff
@@ -145,6 +154,20 @@ function instanceWaitSSH() {
   done
 }
 
+function instanceCheck() {
+  local _ssh="$1"
+
+  # Check if postgres is installed
+  $_ssh rpm -q postgresql
+
+  # Verify subscribe status
+  subscribe_status=$($_ssh subscription-manager status | grep 'Overall Status')
+  [[ ! "$subscribe_status" =~ "Unknown" ]]
```

Comment:
Hm is there a string that we could match rather than a negation? "Any string but something with Unknown" seems a bit too loose of a test :)

Reply:
Change to use "subscription-manager identify". If it's registered, the output is like:
```
system identity: xxxxxx
name: wala84perftest05060901-vm1
org name: 14651757
org ID: 14651757
```
I grep the "org ID" line and compare it with "org ID: $API_TEST_SUBSCRIPTION_ORG_ID" to verify the subscribe status.

If it is not registered, the output is:
```
This system is not yet registered. Try 'subscription-manager register --help' for more information.
```